types/gestalt/index.d.ts

```diff
@@ -1526,7 +1531,12 @@ export interface VideoProps {
     onSeek?: AbstractEventHandler<React.SyntheticEvent<HTMLVideoElement>>;
     onTimeChange?: AbstractEventHandler<React.SyntheticEvent<HTMLVideoElement>, { time: number }>;
     onVolumeChange?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement>, { volume: number }>;
-
+    onError?: AbstractEventHandler<React.SyntheticEvent<HTMLDivElement> | React.SyntheticEvent<HTMLAnchorElement>>;
```

Comment:
I believe this is supposed to be `AbstractEventHandler<React.SyntheticEvent<HTMLVideoElement>`. 

See: https://github.com/pinterest/gestalt/blob/master/packages/gestalt/src/Video.js#L43